README.md

```diff
@@ -65,3 +65,13 @@ The swagger spec will be served at http://localhost:3000/api-docs.json
 ### CLI
 
 You can also use the tool via [command line interface](./docs/CLI.md). It supports selecting multiple files, recursive subdirectories and watch task for continuous listening of changes in your code.
+
+#### CLI Examples
```

Comment:
Better! Only last request from me, please move it to the https://github.com/JesseObrien/swagger-jsdoc/blob/master/docs/CLI.md file which is more specifically for the CLI and just add (optional) to https://github.com/JesseObrien/swagger-jsdoc/blob/master/docs/CLI.md#specify-input-files
Otherwise good work!